echopop/nwfsc_feat/load_data.py

```diff
@@ -303,69 +306,57 @@ def load_geostrata(
 
     if not geostrata_filepath.exists():
         raise FileNotFoundError(f"Geographic stratification file not found: {geostrata_filepath}")
-    
+
     # First load each sheet
     raw_geostrata_dict = {
-        strata_type: load_single_stratum_sheet(
-            geostrata_filepath, sheet_name, column_name_map
-        )
+        strata_type: load_single_stratum_sheet(geostrata_filepath, sheet_name, column_name_map)
         for strata_type, sheet_name in geostrata_sheet_map.items()
     }
-    
+
     # Then process each dataframe with the extracted function
     processed_geostrata_dict = {
-        strata_type: preprocess_geostrata_data(df)
-        for strata_type, df in raw_geostrata_dict.items()
+        strata_type: preprocess_geostrata_data(df) for strata_type, df in raw_geostrata_dict.items()
     }
-    
+
     return processed_geostrata_dict
 
-def preprocess_geostrata_data(
-    df: pd.DataFrame
-) -> pd.DataFrame:
+
+def preprocess_geostrata_data(df: pd.DataFrame) -> pd.DataFrame:
     """
```

Comment:
I think this function should be absorbed into `load_geostrata` since the content is short and this function is only used once.